frontend/src/scenes/error-tracking/errorTrackingSymbolSetLogic.tsx

```diff
@@ -70,17 +77,29 @@ export const errorTrackingSymbolSetLogic = kea<errorTrackingSymbolSetLogicType>(
 
     forms(({ values, actions }) => ({
         uploadSymbolSet: {
-            defaults: { files: [] } as { files: File[] },
-            submit: async ({ files }) => {
-                if (files.length > 0 && values.uploadSymbolSetReference) {
-                    const formData = new FormData()
-                    const file = files[0]
-                    formData.append('source_map', file)
-                    await api.errorTracking.updateSymbolSet(values.uploadSymbolSetReference, formData)
-                    actions.setUploadSymbolSetReference(null)
-                    actions.loadSymbolSets()
-                    lemonToast.success('Source map uploaded')
+            defaults: { minified: [], sourcemap: [] } as { minified: File[]; sourcemap: File[] },
```

Comment:
Source map is two separate words
```suggestion
            defaults: { minified: [], sourceMap: [] } as { minified: File[]; sourceMap: File[] },
```